tg.js

```diff
@@ -1,821 +1,821 @@
+windows.controls.kickVoteY Tanggalin mula sa laro [Oo]
+windows.controls.kickVoteN Tanggalin mula sa laro [Hindi]
+windows.controls.kpdVoteY Vote sa KPD [Oo]
+windows.controls.kpdVoteN Vote sa KPD [Hindi]
+windows.controls.specFree I-toggle ang Free Cam
+windows.controls.specObj I-toggle ang Objective Cam
+windows.controls.specFirst I-toggle ang First Person
+windows.controls.specNames I-toggle ang mga pangalan
+windows.controls.streak0 Streak 1
+windows.controls.streak1 Streak 2
+windows.controls.streak2 Streak 3
+windows.controls.streak3 Streak 4
+windows.controls.streak4 Streak 5
+windows.controls.premium0 Naka noclip
+windows.controls.premium1 Naka godmode
+windows.controls.premium2 Pumatay
+windows.controls.premium3 Hampasin
+windows.controls.dev0 Bind ng Dev 1
+windows.controls.dev1 Bind ng Dev 2
+windows.controls.dev2 Bind ng Dev 3
+windows.controls.dev3 Bind ng Dev 4
+windows.controls.reset Balik-Simula ang posisyon (Paligsahan at Parkour)
+windows.controls.pretend Itago ang Laro
+windows.host.header Host ng Custom na Laro
+windows.host.maps Pag-setup ng Server
+windows.host.cmaps Laro mula sa Komunidad
+windows.host.raw Raw Data ng Laro
+windows.host.modes Piliin ang Uri ng Laro 
+windows.host.classes Piliin ang mga Gagamiting Uri ng Armas
+windows.host.settings Mga Settings ng Server
+windows.host.preset Handang Settings ng Host
+windows.host.preset.load Loading ang mga hinandang settings...
+windows.host.preset.name Preset Pangalan
+windows.host.start Magsimula
+windows.host.rehost Magsimula Muli ng Laro (Magdala ng mga Manlalaro)
+windows.spray.header Pillin ang Sspray
+windows.spray.default Pumili ng Karaniwang Spray
+windows.spray.spin Umikot para sa Spray
+windows.maps.header Mga Laro at Mapa
+windows.maps.none Walang nahanap na Laro!
+windows.maps.name Pangalan ng Laro
+windows.maps.editor Game Maker
+windows.maps.publish I-publish at I-akda ang Laro
+windows.maps.community Komunidad ng Devs
+windows.publish.map.header I-publish o I-update ang Laro
+windows.publish.map.thumbnail Walang Thumbnail o Larawan
+windows.publish.map.paste I-paste ang Data ng Laro
+windows.publish.map.update Publish/Update
+windows.publish.map.login Lumikha at I-upload ang laro!
+windows.theatre.header Modo na Theatre
+windows.theatre.record Paganahin ang Pagre-record
+windows.theatre.key Pang-record
+windows.clans.header Samahan
+windows.clans.members Mga Miyembro ({0})
+windows.clans.requests Mga Nais Sumali 
+windows.clans.requests.none Walang Nais Sumali
+windows.clans.page [{0}] Pahina ng Samahan
+windows.clans.create Lumikha ng Samahan
+windows.clans.name Pangalan ng Samahan
+windows.clans.view Tignan ang mga Samahan
+windows.clans.join Sumali sa Samahan
+windows.clans.login Mag-login upang makagawa o makasali sa isang Samahan!
+windows.store.header Tindahan
+windows.store.spin {0} Premyo
+windows.store.agree Sa pamamagitan ng pagbili sumasang-ayon ka sa {0} Mga Tuntunin
+windows.store.purchase Bumili ng KR
+windows.store.voucher Kunin ang voucher

```

Comment:
```suggestion
windows.store.voucher Gamitin ang voucher
```
Using kunin in this context would translate to "take the voucher"